### PR TITLE
Basic behave testing for sessions 4/5 - PMT #100729

### PR DESCRIPTION
**Pull request by nikolas**
This includes a time.sleep(1) call in one of the steps to try
and avoid the intermittent behave failure we saw on travis earlier
today.

### CODE REVIEW

File: features/steps/common.py [hunk 1]
```diff
@@ -27,6 +28,7 @@ def i_click_the_first_participant_journal_button(context):
 
 @when(u'I click the link "{text}"')
 def i_click_the_link(context, text):
+    time.sleep(1)
```
Comment (sdreher) [inline features/steps/common.py]: @thraxil has a good pattern for timing related failures. The approach is to try the test first, catch the exception, sleep and retry. This circumvents slowdowns when the initial attempt is likely to succeed. 
Example: https://github.com/ccnmtl/mediathread/blob/master/terrain.py#L304

Comment (nikolas) [inline features/steps/common.py]: Thanks - that's cool. I added that.

**Comment by thraxil** [comment]
tbh, I'm not totally behind that pattern. It works, and sometime's it's the only option, or at least the only option that doesn't involve re-architecting other components entirely. I still kind of feel dirty using a `sleep()` to "fix" a concurrency problem.

This is worth perusing: http://www.obeythetestinggoat.com/how-to-get-selenium-to-wait-for-page-load-after-a-click.html

**Comment by sdreher** [comment]
Agreed on the sleep. Seems like just choosing the least evil path.

I have been experimenting with the WebDriveWait.until methods with some success. https://github.com/ccnmtl/mediathread/blob/master/mediathread/taxonomy/features/taxonomy-steps.py#L151

**Comment by thraxil** [comment]
Heads up that I merged this, but it looks like it still intermittently fails on Travis.

**Pull request merged**